### PR TITLE
Update Sbom targets E2E tests to .net8

### PR DESCRIPTION
**Pull request by vpatakottu**
Added some NuGet packages that were required to make the E2E tests compatible with .NET8

### CODE REVIEW

File: Directory.Packages.props [hunk 1]
```diff
@@ -1,63 +1,65 @@
...
-        <PackageVersion Include="System.Reactive" Version="5.0.0" />
-        <PackageVersion Include="System.Runtime.Loader" Version="4.3.0" />
-        <PackageVersion Include="System.Text.Json" Version="8.0.4" />
-        <PackageVersion Include="System.Threading.Channels" Version="6.0.0" />
-        <PackageVersion Include="System.Threading.Tasks.Dataflow" Version="4.11.1" />
-        <PackageVersion Include="System.Threading.Tasks.Extensions" Version="4.5.4" />
-    </ItemGroup>
-</Project>
+  <ItemDefinitionGroup>
+    <PackageVersion>
```
Comment (DaveTryon) [inline Directory.Packages.props:4]: I'm not sure why this file uses 4-spaces for each indent, but let's maintain that if possible. If/when we decide to change the indents, let's put that into a separate PR.

Comment (vpatakottu) [reply Directory.Packages.props]: Updated to fix the indents!
